### ISSUE TITLE
Tech Stack

### ISSUE THREAD

**Issue by dev-avocado** [dev-avocado/progress-tracking-system#2]
This will be a web application either in one of these two tech stacks, based on the contributors who are willing to participate:
1. MERN Stack(MongoDB, Express.js, React.js, Node.js)
2. HTML/CSS/JavaScript + PHP/MySQL

If anyone is willing to work on one of these two stacks in froontend/backend, comment on this issue.

**Comment by Debajyoti111** [comment]
Hi @dev-avocado I am a mern stack developer and am interested in working on this issue! But first can you please elaborate on this?

**Comment by dev-avocado** [comment]
Hey @Debajyoti111 

Trying to put the idea in a  simpler way, this would be a web app, which helps people track their progress for a particular set of tasks, but this is majorly focused on learning courses, etc. 

One can create a list(similar to a todo list), for eg.

List name: Maths

Topics:

- [ ] Differential equations
- [ ] Integrations
- [x] Limits
- [ ] Statistics

Now, when the person check marks Limits, then 25% of his todo list is completed(1 task out of 4 is done), so the app should show in the progress, 25% done:

![image](https://user-images.githubusercontent.com/114883179/193902232-4f5e8323-4704-4e0b-a6ed-a94727626378.png)

(above image is how it will look - this isn't the design but just the representation of feature for better understanding)

We are also planning to add **Streak**, **Notifications** and other such features, because this gives the user a sense of accomplishment and motivates them to do more, and Signup/login to save their progress.


Initially, we're trying to keep it simple, these are the features we wish to have in this app, and currently working on:

1. Signup/Login
2. Fixed list of tasks(we will add CRUD later, once the basic functionalities are working)
3. Dashboard with sidebar
4. Progress bar

**Comment by dev-DTECH** [comment]
I am Interested in working on this issue

**Comment by dev-avocado** [comment]
Hey @dev-DTECH   
You can work on it. We can have a small discussion on which stack we should work.

**Comment by iamkennis** [comment]
I am Interested in working on this issue

**Comment by dev-DTECH** [comment]
> Hey @dev-DTECH You can work on it. We can have a small discussion on which stack we should work.

Yeah sure so basically I am well acquainted with mern stack but will be interested to work on the frontend.
Also, I wanted to ask will this repo be considered in hacktoberfest?

**Comment by dev-avocado** [comment]
> 

Welcome, which stack are you familiar with, HTML/CSS or MERN(ReactJs)?

**Comment by dev-avocado** [comment]
Welcome @iamkennis
Which stack are you familiar with, HTML/CSS or MERN(ReactJs)?

**Comment by dev-avocado** [comment]
> > Hey @dev-DTECH You can work on it. We can have a small discussion on which stack we should work.
> 
> Yeah sure so basically I am well acquainted with mern stack but will be interested to work on the frontend. Also, I wanted to ask will this repo be considered in hacktoberfest?

No, it won't be.

**Comment by iamkennis** [comment]
Am familiar with MERN Stack

**Comment by dev-avocado** [comment]
@dev-DTECH @iamkennis @Debajyoti111 

Please join our [discord server](https://discord.gg/WGFTjCtk) for better communication
